### PR TITLE
#6487 qemu-test.json template, #6488 qemu nodes generator, #6489 role…

### PR DESCRIPTION
**Pull request by h0st**

### CODE REVIEW

File: core/generator.rb [hunk 1]
```diff
@@ -324,7 +360,7 @@ def Generator.nodeDefinition(node, boxes, path, cookbook_path)
   return machine
 end
 
-def Generator.generate(path, config, boxes, override, aws_config)
+def Generator.generate(path, config, boxes, override, aws_config, provider)
```
Comment (krinkin) [inline core/generator.rb:375]: looks like aws_config should be moved to $session

Comment (h0st) [inline core/generator.rb:375]: aws_config moved to $session

**Comment by krinkin** [comment]
Update with master. It has conflicts

**Comment by h0st** [comment]
Fixed

**Comment by h0st** [comment]
Fixed twice

**Pull request merged**